drivers/sensor/tmp112/tmp112.c

```diff
@@ -194,6 +194,41 @@ static int tmp112_channel_get(const struct device *dev,
 	return 0;
 }
 
+#ifdef CONFIG_PM_DEVICE
+static int tmp112_pm_control(const struct device *dev, uint32_t command,
+			     uint32_t *state, pm_device_cb cb, void *arg)
+{
+	int ret = 0;
+	struct tmp112_data *device_data = get_data(dev);
+
+	switch (command) {
+	case PM_DEVICE_STATE_SET:
+		if (*state == PM_DEVICE_STATE_ACTIVE
+		    && device_data->power_state
+		    == PM_DEVICE_STATE_SUSPEND) {
+			tmp112_update_config(dev, 0, 0);
+			device_data->power_state = PM_DEVICE_STATE_ACTIVE;
+		} else if (*state == PM_DEVICE_STATE_SUSPEND) {
+			device_data->power_state = PM_DEVICE_STATE_SUSPEND;
+		} else {
+			ret = -EINVAL;
+		}
+		break;
+	case PM_DEVICE_STATE_GET:
+		*state = device_data->power_state;
+		break;
+	default:
+		ret = -EINVAL;
+	}
+
+	if (cb) {
+		cb(dev, ret, state, arg);
+	}
+
+	return ret;
+}
+#endif
+
```

Comment:
I was expecting tmp112_pm_control to be part of the driver definition.

Reply:
done